docs/en/developers/03_Running_tests.md

```diff
@@ -34,3 +34,43 @@ They are performed with [GitHub Actions](https://github.com/FreshRSS/FreshRSS/ac
 This ensures your code will not introduce some kind of regression. We will not merge a PR if tests fail so we will ask you to fix any bugs before reviewing your code.
 
 If you are interested, you can take a look at [the configuration file](https://github.com/FreshRSS/FreshRSS/blob/edge/.github/workflows/tests.yml).
+
+## Using feed snapshots
+
+As feed data is volatile, it’s better to work with snapshots when debugging some issues.
+You can find the description to retrieve a snapshot [here](06_Reporting_Bugs.md#how-to-provide-feed-data).
+
+To serve those snapshots, you can use a mock server.
+Here we will demonstrate how to work with [WireMock](https://wiremock.org/) but other solutions exist.
```

Comment:
Interesting. For the record, I'd intuitively probably be inclined to do something like:

```
docker run -p 80:80 -v $(pwd):/usr/share/nginx/html nginx
```

Which is basically the same thing but without the WireMock part. ;-)

Reply:
Or `python3 -m http.server 8080` :)

Reply:
Or `python3 -m http.server 8080` :)

Reply:
I haven't though on that but those are valid suggestions. Do you think I should add them or completely disregard the wiremock part?

Reply:
I do believe both suggestions above are an order of magnitude simpler to use compared to messing with JSON for the vast majority of use cases.

* Name your file lala.rss → content-type: application/rss+xml
* Name it asdf.atom → content-type: application/atom+xml

So you can also easily test incorrect mimetypes with e.g. .txt and .html.

Reply:
In that case, I will remove that section and only mention few ways one can achieve that.